cfme/common/vm.py

```diff
@@ -522,20 +521,19 @@ class VM(BaseVM):
     TO_RETIRE = None
 
     retire_form = Form(fields=[
-        ('date_retire', {
-            version.LOWEST: date_retire_element,
-            "5.5": AngularCalendarInput(
-                "retirement_date", "//label[contains(normalize-space(.), 'Retirement Date')]")}),
+        ('date_retire', AngularCalendarInput(
+            "retirement_date", "//label[contains(normalize-space(.), 'Retirement')]")),
```

Comment:
So, check out AngularCalendarInput, the second argument is a 'click away element'. It's clicked after the calendar element itself is interacted with on a fill, to ensure the calendar is closed.

I think in this case its fine that the locator matches two labels on the page, clicking either one will produce the desired result.